symfony/usr/local/share/env/45-framework

```diff
@@ -10,3 +10,4 @@ fi
 
 export SYMFONY_MAJOR_VERSION=${SYMFONY_MAJOR_VERSION:-${SYMFONY_MAJOR_VERSION_GUESS}}
 export SYMFONY_CONSOLE=${SYMFONY_CONSOLE:-${SYMFONY_CONSOLE_DEFAULT}}
+export SYMFONY_APP_ENDPOINT_REGEX=$(php -r 'echo preg_quote(getenv("SYMFONY_APP_ENDPOINT"));')
```

Comment:
I've checked and ENV_VAR=$(echo test test) and ENV_VAR=${VAR:-test test} appear to still treat the value as one argument, so the quotes aren't needed